pandas/tests/io/json/test_pandas.py

```diff
@@ -1895,6 +2001,7 @@ def test_to_s3(self, s3_public_bucket, s3so):
             timeout -= 0.1
             assert timeout > 0, "Timed out waiting for file to appear on moto"
 
+    @pytest.mark.filterwarnings("ignore::FutureWarning")
```

Comment:
Can you follow the same pattern you applied with other code samples? The marker is more ambiguous as to why we need to filter

Reply:
The reason I did this is because only some of the fixtures had a datetimelike value, and I don't know how to filter them out besides using the same check I did in `to_json` (or hard coding) which I didn't think is appropriate

Let me know if you still think I should do it

Reply:
Agreed it's better to use use `assert_produces_warning` in the test to validate the warning under certain scenarios than use `filterwarnings`